go/genkit/flow.go

```diff
@@ -285,28 +303,22 @@ func (f *Flow[In, Out, Stream]) runInstruction(ctx context.Context, inst *flowIn
 	}
 }
 
-// flow is the type that all Flow[In, Out, Stream] have in common.
-type flow interface {
-	Name() string
-
-	// runJSON uses encoding/json to unmarshal the input,
-	// calls Flow.start, then returns the marshaled result.
-	runJSON(ctx context.Context, input json.RawMessage, cb streamingCallback[json.RawMessage]) (json.RawMessage, error)
-}
+// Flow[I, O, S] implements the flow interface.
```

Comment:
Should this comment live up at the definition of Flow?
Or do you mean here "The following methods ensure that ... "

Reply:
The latter. Done.